CloudFormation/arkcase.yml

```diff
@@ -1452,16 +1543,530 @@ Resources:
+  #   Properties:
+  #     Port: 443
+  #     Protocol: HTTPS
+  #     VpcId: !Ref Vpc
+  #     TargetType: ip
+  #     HealthCheckEnabled: true
+  #     HealthCheckIntervalSeconds: 10
+  #     HealthCheckProtocol: HTTPS
+  #     HealthCheckPath: /alfresco/
+  #     HealthCheckTimeoutSeconds: 5
+  #     HealthyThresholdCount: 2
+  #     UnhealthyThresholdCount: 2
+
+  # AlfrescoListenerHttp:
+  #   Type: AWS::ElasticLoadBalancingV2::Listener
+  #   Properties:
+  #     LoadBalancerArn: !Ref AlfrescoLoadBalancer
+  #     Port: 80
+  #     Protocol: HTTP
+  #     DefaultActions:
+  #       - Type: redirect
+  #         RedirectConfig:
+  #           Host: '#{host}'
+  #           Path: '/#{path}'  # yamllint disable-line rule:quoted-strings
+  #           Query: '#{query}'
+  #           Port: 443
+  #           Protocol: HTTPS
+  #           StatusCode: HTTP_302
+
+  # AlfrescoListenerHttps:
+  #   Type: AWS::ElasticLoadBalancingV2::Listener
+  #   Properties:
+  #     LoadBalancerArn: !Ref AlfrescoLoadBalancer
+  #     Port: 443
+  #     Protocol: HTTPS
+  #     DefaultActions:
+  #       - Type: forward
+  #         TargetGroupArn: !Ref AlfrescoRepositoryTargetGroup
+  #     Certificates:
+  #       - CertificateArn: !GetAtt Pki.Outputs.AlfrescoIamCertArn
+  #     SslPolicy: ELBSecurityPolicy-FS-1-2-Res-2019-08
+
+  # Route53 for Alfresco
+
+  # AlfrescoDnsRecord:
+  #   Type: AWS::Route53::RecordSet
+  #   Properties:
+  #     Name: !Sub alfresco.${Project}.lan
+  #     Type: A
+  #     HostedZoneId: !Ref PrivateHostedZone
+  #     AliasTarget:
+  #       DNSName: !GetAtt AlfrescoLoadBalancer.DNSName
+  #       EvaluateTargetHealth: false
+  #       HostedZoneId: !GetAtt AlfrescoLoadBalancer.CanonicalHostedZoneID
+
+  # EFS filesystem
+
+  AlfrescoEfsMountTargetSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
     Properties:
-      Name: !Sub test-backend.${Project}.lan
-      Type: A
-      HostedZoneId: !Ref PrivateHostedZone
-      AliasTarget:
-        DNSName: !GetAtt TestBackendLoadBalancer.DNSName
-        EvaluateTargetHealth: false
-        HostedZoneId: !GetAtt TestBackendLoadBalancer.CanonicalHostedZoneID
+      GroupDescription: Security group for the Alfresco EFS mount targets
+      VpcId: !Ref Vpc
+      SecurityGroupIngress:
+        - Description: NFSv4 from this VPC
+          IpProtocol: tcp
+          FromPort: 2049
+          ToPort: 2049
+          SourceSecurityGroupId: !GetAtt AlfrescoTaskSecurityGroup.GroupId
+      Tags:
+        - Key: Name
+          Value: !Sub alfresco-efs-security-group-${Project}-${Env}
+        - Key: Env
+          Value: !Ref Env
+        - Key: Project
+          Value: !Ref Project
+        - Key: Service
+          Value: alfresco
+        - Key: Component
+          Value: vpc
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  AlfrescoEfs:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      Encrypted: true
+      FileSystemTags:
```

Comment:
EFS by default has pretty low (but cheap) bandwidth.  If we see some performance issue we may want to enable provisioned bandwidth, or we may want to add a parameter as to whether to enable provisioned bandwidth.